components/centraldashboard/cypress/e2e/main-page.cy.ts

```diff
@@ -27,27 +27,20 @@ describe('Main Page', () => {
     cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Quick-Links').should('exist');
     cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Quick-Links').find('iframe-link').should('exist').and('have.length', 1).and('have.prop', 'href', '/en/new?ns=test-namespace');
     cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Quick-Links').find('iframe-link').find('div.header').should('have.text', 'Create a new Notebook server');
-    // recent notebooks links
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('notebooks-card').should('exist');
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('notebooks-card').shadow().find('iframe-link').should('have.length', 5);
-    // documentation links
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Documentation').should('exist');
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Documentation').find('a').should('have.length', 4);
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Documentation').find('a').eq(0).should('have.prop', 'href', 'https://statcan.github.io/aaw/');
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Documentation').find('a').eq(0).find('div.header').should('have.text', 'Advanced Analytics Workspace Docs');
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Documentation').find('a').eq(0).find('aside').should('have.text', 'Helpful guides about our data and analysis tools');
-
-    cy.get('main-page').shadow().find('dashboard-view').shadow().find('paper-card#Documentation').find('a').eq(1).should('have.prop', 'href', 'https://www.youtube.com/playlist?list=PL1zlA2D7AHugkDdiyeUHWOKGKUd3MB_nD');
```

Comment:
Here, you removed the check for the documentations links at `.find('a').eq(1)`, so now it would only check for the first,third and forth links. 

Also, not sure why you moved down the checks for  at `.find('a').eq(0)` that had the `// documentation links` comment above it. You should move that back, or else the comment starts being innacurate since its suppose to denote that what is underneath is the testing for documentation links.